README.md

```diff
@@ -404,6 +404,35 @@ GET /_opendistro/_knn/HYMrXXsBSamUkcAjhjeN0w/stats/circuit_breaker_triggered,gra
 }
 ```
 
+## Warmup API
+### Overview
+The HNSW graphs used to perform k-Approximate Nearest Neighbor Search are stored as `.hnsw` files with the other Lucene segment files. In order to perform search on these graphs, they need to be loaded into memory. If the graphs have not yet been loaded into memory, upon search, they will first be loaded and then searched. This can cause high latency during initial queries. To avoid this, users will often run random queries during a warmup period. After this warmup period, the graphs will be loaded into memory and their production workloads can begin. This process is indirect and requires extra effort. 
```

Comment:
Minor: In places where we mention ```memory``` should we be explicit calling ```native memory``` or ```off-heap``` to set context where the graphs are actually loaded?

Reply:
Sure, will update